client/src/components/relatedProducts/relatedProductsList.jsx

```diff
@@ -4,18 +4,23 @@ import Carousel from 'react-multi-carousel';
 import "react-multi-carousel/lib/styles.css";
 import axios from 'axios';
 import API_KEY from '../../../../config.js'
+import Card from './card.jsx'
 // import styled from 'styled-components';
 
+function sum(a, b) {
```

Comment:
maybe create a more descriptive name for this function. What is it summing? What is the purpose?

Reply:
the function was deleted later on. It was just a test function in though i might use in jest test file but didnt use.